rktl_sim/src/simulator/sim.py

```diff
@@ -74,20 +89,37 @@ def __init__(self, props, urdf_paths, spawn_bounds, render_enabled):
 
         if 'engine' in self.props and self.props['engine'] is not None:
             p.setPhysicsEngineParameter(**self.props['engine'])
+        # TODO: why gravity not accurate?
```

Comment:
See previous comment about TODOs. This probably should be parameterized, it's been solid up to this point, but we want to likely include that in domain randomization.